staff/e2e/routes/index.spec.ts

```diff
@@ -0,0 +1,19 @@
+// index.spec.ts - tests for the index page
+import { test, expect } from "@playwright/test";
+import AxeBuilder from "@axe-core/playwright";
+
+// test("index has no automatically detectable accessibility errors", async ({
+//   page,
+// }) => {
+//   await page.goto("/");
+//   const accessibilityScanResults = await new AxeBuilder({ page }).analyze();
+//   expect(accessibilityScanResults.violations).toEqual([]);
+// });
```

Comment:
👀

Reply:
I commented this out because I didn't want to improve the accessibility of the default lowdefy index page. I'll turn this back on for our actual pages.